ios/FluentUI.Demo/FluentUI.Demo/Demos/SideBarDemoController.swift

```diff
@@ -0,0 +1,122 @@
+			TabBarItem(title: "New", image: UIImage(named: "New_28")!, selectedImage: UIImage(named: "New_Selected_28")!, landscapeImage: UIImage(named: "New_24")!, landscapeSelectedImage: UIImage(named: "New_Selected_24")!),
+			TabBarItem(title: "Open", image: UIImage(named: "Open_28")!, selectedImage: UIImage(named: "Open_Selected_28")!, landscapeImage: UIImage(named: "Open_24")!, landscapeSelectedImage: UIImage(named: "Open_Selected_24")!)
+		]
+
+		sideBar!.bottomItems = [
+			TabBarItem(title: "Help", image: UIImage(named: "Help_28")!, selectedImage: UIImage(named: "Help_Selected_28")!, landscapeImage: UIImage(named: "Help_24")!, landscapeSelectedImage: UIImage(named: "Help_Selected_24")!),
+			TabBarItem(title: "Settings", image: UIImage(named: "Settings_28")!, selectedImage: UIImage(named: "Settings_Selected_28")!, landscapeImage: UIImage(named: "Settings_24")!, landscapeSelectedImage: UIImage(named: "Settings_Selected_24")!)
+		]
+
+		let optionsStackView = UIStackView(frame: .zero)
+		optionsStackView.axis = .vertical
+		optionsStackView.alignment = .center
+		optionsStackView.spacing = 5.0
+		optionsStackView.translatesAutoresizingMaskIntoConstraints = false
+		controller.view.addSubview(optionsStackView)
+
+		let showAvatarViewRow = createLabelAndSwitchRow(labelText: "Show Avatar View", switchAction: #selector(toggleAvatarView(switchView:)), isOn: true)
+		showAvatarViewRow.translatesAutoresizingMaskIntoConstraints = false
+		optionsStackView.addArrangedSubview(showAvatarViewRow)
+		showAvatarView(true)
+
+		let button = Button()
+		button.translatesAutoresizingMaskIntoConstraints = false
+		button.titleLabel?.textAlignment = .center
+		button.titleLabel?.numberOfLines = 0
+		button.setTitle("Dismiss", for: .normal)
+		button.addTarget(self, action: #selector(dismissSideBar), for: .touchUpInside)
+		optionsStackView.addArrangedSubview(button)
+
+		NSLayoutConstraint.activate([
+			optionsStackView.centerXAnchor.constraint(equalTo: controller.view.centerXAnchor),
+			optionsStackView.centerYAnchor.constraint(equalTo: controller.view.centerYAnchor),
+		])
+
+		present(controller, animated: false)
+	}
+
+	private func createLabelAndSwitchRow(labelText: String, switchAction: Selector, isOn: Bool = false) -> UIView {
+		let stackView = UIStackView(frame: .zero)
+		stackView.axis = .horizontal
+		stackView.alignment = .center
+		stackView.spacing = 10.0
+
+		let label = Label(style: .subhead, colorStyle: .regular)
+		label.text = labelText
+		stackView.addArrangedSubview(label)
+
+		let switchView = UISwitch()
+		switchView.isOn = isOn
+		switchView.addTarget(self, action: switchAction, for: .valueChanged)
+		stackView.addArrangedSubview(switchView)
+
+		return stackView;
+	}
+
+	@objc private func dismissSideBar() {
+		dismiss(animated: false, completion: nil)
+	}
+
+	@objc private func toggleAvatarView(switchView: UISwitch) {
+		showAvatarView(switchView.isOn)
+	}
+
+	private func showAvatarView(_ show: Bool) {
+		var avatarView: AvatarView?
```

Comment:
Another minor thing: can we avoid a dependency on AvatarView here? There is a goal to make fluentui-apple "composable" by each client separately so they can only take the pieces they need and reduce overall bundle size impact. If we can force each component to be "independently" testable, that will help

Reply:
I originally designed this to take any view and then switched to avatar view.
The intention is to force a specific design for this control. The user MUST show an avatar view here. We don't want to give the flexibility of passing in any view here because we can't control how this will look.
The goal of this control library is to provide a unified set of controls and a unified design. We wouldn't apps to start placing in custom views here. We would want everyone to adopt avatar view and use that to display the user image. Therefore, we have a hard dependency on avatar view in this component.